README.md

```diff
@@ -17,9 +17,14 @@ If you want/need to build, read on.
 ## Before you build this site
 
 ### 1. Get the prerequisites
+Flutter 1.12.13 - Latest Version 
```

Comment:
Is this something we can reasonably maintain? We could just do "Flutter 1.12" instead.